keras_lr_finder/lr_finder.py

```diff
@@ -39,7 +40,7 @@ def on_batch_end(self, batch, logs):
 
     def find(self, x_train, y_train, start_lr, end_lr, batch_size=64, epochs=1):
         num_batches = epochs * x_train.shape[0] / batch_size
-        self.lr_mult = (float(end_lr) / float(start_lr)) ** (float(1) / float(num_batches))
+        self.lr_mult = (float(end_lr) / float(start_lr)) ** (1.0 / float(num_batches)*float(epochs))
```

Comment:
Maybe some parentheses are missing? Now it's dividing 1 by num_batches and then multiplying by epochs, which is effectively dividing by (num_batches / epochs). I think the last "*" should be replaced with "/", or parentheses added.

Reply:
I changed the wrong function, my bad. My fix should only be concerned with the generator method, not the regular one.